src/util/getEventHandlers.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path')
+const glob = require('glob')
+
+const messagesDir = path.dirname(require.resolve('~/src/messages'))
+
+const eventHandlers = {}
+let initialized = false
+
+// create a map of message types to handler functions
+module.exports = function getEventHandlers () {
+  if (!initialized) {
```

Comment:
This is not actually necessary if you just initialize `eventHandlers` as `undefined` and then just check to see if it's falsy.